### PR TITLE
[RM README] Update Status and TOC with Appendix

### LINKED ISSUE #2769: [RM README] Link to Appendix in RM TOC

### PR DESCRIPTION
**Pull request by pgoyal01**
Fixes #2769

### CODE REVIEW

File: doc/ref_model/README.md [hunk 1]
```diff
@@ -38,6 +38,8 @@
 | Chapter 08 | Release work finished |
 | Chapter 09 | Release work finished |
 | Chapter 10 | Release work finished |
+| Chapter 10 | Release work finished |
```
Comment (ulikleber) [inline doc/ref_model/README.md:41]: now two chapter 10?

**Comment by walterkozlowski** [comment]
@pgoyal01, can you please correct and we will approve and merge.  Thanks!

File: doc/ref_model/README.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_model/README.md]: @ulikleber @karinesevilla Thanks - Done

**Comment by pgoyal01** [comment]
@walterkozlowski Fixed issues and conflicts. Thanks

**Review by walterkozlowski — APPROVED** [review]

**Pull request merged**